### DISCUSSION TITLE
Unable to update codelist

### DISCUSSION THREAD

**Discussion by rc16** [opensafely/documentation#904] [Codelists] [answered]
Hi,

I'm working locally on this repo: https://github.com/opensafely/RA_outcomes 

I've updated the codelists text file and am trying to run opensafely codelists update but I am getting the error "TBC does not match [project]/[codelist]/[version] or user/[username]/[codelist]/[version]" and I'm not sure why?

I can run other actions on the yaml file e.g. generate_study_population, I can also update the codelist in another repo.

Ruth

**Comment by iaindillingham** [comment, marked as answer]
Hello Ruth,

I think you're working from the `hospitalisations` branch. You'll notice the following line in *codelists/codelists.txt*:

```
# Opioids
TBC
```

This is the "TBC" that is referenced by the error message. You could either remove it or comment it.

**Reply by rc16** [reply]
Ah I hadn't noticed that, thank you!
